Maestro/subscriptions.json

```diff
@@ -514,6 +514,7 @@
         "https://github.com/dotnet/coreclr/blob/release/**/*",
         "https://github.com/dotnet/core-setup/blob/master/**/*",
         "https://github.com/dotnet/core-setup/blob/release/**/*",
+        "https://github.com/dotnet/diagnostics/blob/master/**/*",
```

Comment:
I've added mirroring for the dotnet.visualstudio.com instance.  This should be moved into the dotnet vsts mirroring list.

Reply:
@mmitche Should just close this PR?

Reply:
Yeah